src/isort/settings.rs

```diff
@@ -99,9 +111,23 @@ impl Settings {
             force_wrap_aliases: options.force_wrap_aliases.unwrap_or_default(),
             known_first_party: BTreeSet::from_iter(options.known_first_party.unwrap_or_default()),
             known_third_party: BTreeSet::from_iter(options.known_third_party.unwrap_or_default()),
+            order_by_type: options.order_by_type.unwrap_or_default(),
```

Comment:
I think this should be `options.order_by_type.unwrap_or(true)`. As-is, it will default to `false`.